pkg/exchange/okex/parse.go

```diff
@@ -38,7 +38,7 @@ type WebSocketEvent struct {
 	Arg     interface{} `json:"arg,omitempty"`
 }
 
-func parseEvent(v *fastjson.Value) (*WebSocketEvent, error) {
+func parseEvent(v *fastjson.Value) (*WebSocketEvent, string, error) {
```

Comment:
why returning the event separately here? it's already included in the event object.

Reply:
i was thinking to make the interface consistency, haha.  i have implemented this in another way now.